spec/menu_spec.rb

```diff
@@ -0,0 +1,13 @@
+require './docs/menu'
+
+describe Menu do
+
+  it 'read menu' do
+    menu = {:fish_burger => 2.0,
+            :beef_burger => 4.0,
+            :chicken_burger => 4.0,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)